toolchain/src/integration-test/java/org/archcnl/toolchain/ToolchainIT.java

```diff
@@ -0,0 +1,92 @@
+package org.archcnl.toolchain;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+import org.apache.jena.ontology.OntModel;
+import org.apache.jena.ontology.OntModelSpec;
+import org.apache.jena.query.Query;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryExecutionFactory;
+import org.apache.jena.query.QueryFactory;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.archcnl.stardogwrapper.api.StardogDatabaseAPI;
+import org.archcnl.stardogwrapper.impl.StardogDatabase;
+import org.junit.Test;
+
+public class ToolchainIT {
+
+    private final String rootDir = "./src/integration-test/resources/";
+    private final String database = "archcnl_it_db";
+    private final String server = "http://localhost:5820";
+    private final String username = "admin";
+    private final String password = "admin";
+    private final String context = "http://graphs.org/" + database + "/1.0";
+    private final String resultPath = rootDir + "result.owl";
+    
+    @Test
+    public void testArchcnl() throws IOException {
+        CNLToolchain.runToolchain(database,
+                server,
+                context,
+                username,
+                password,
+                Arrays.asList(rootDir + "OnionArchitectureDemo/src/"), // source paths
+                rootDir + "OnionArchitectureDemo/rules.adoc", // rule file 
+                false, // verbose logging
+                true); // remove previous databases
```

Comment:
This needs a rebase onto the current master as `runToolchain()` has got another parameter `enabledParsers`.

Reply:
:+1: